frontend/src/components/TaskMap.module.scss

```
@@ -7,16 +7,15 @@ $ZOOM: var(--zoom, 1);
   width: 100%;
 }
 
-.taskName {
-  width: 210px;
-  max-height: 40px;
-  text-align: left;
-  overflow: hidden;
-  word-wrap: break-word;
+.info {
+  position: relative;
+  left: 28px;
+  bottom: 26px;
+  width: 16px;
+}
 
-  -webkit-box-orient: vertical;
-  display: -webkit-box;
-  -webkit-line-clamp: 2;
+.taskName {
+  @extend .taskName;
 
   &.dragging {
```

I'm not too familiar with the name resolution/scoping and how exactly it would work here,
but even if this works, I'd pick different names just for clarity

Or would it be sufficient to add just the dragging rule? (the dragging styles aren't exactly needed for the measuring)

```suggestion
.taskName.dragging {
```

---

Valid point even though it works in development. Let's go with a different name for clarity